src/resources/locales/en.json

```diff
@@ -111,14 +111,19 @@
         "addMenu": "Add OPDS feed",
         "back": "Back",
         "breadcrumbRoot": "Catalogs",
-        "empty": "",
+        "empty": "No publication found",
         "menu": {
             "aboutBook": "About this book",
             "addExtract": "Import the extract",
             "addTeaser": "Add the extract to the bookshelf",
             "goBuyBook": "Buy this book",
             "goLoanBook": "Borrow this book",
             "goSubBook": "Subscribe"
+        },
+        "network": {
+            "error": "Unable to access to opds feed",
+            "reject": "An error has occurred",
```

Comment:
Given the catch-all nature of this error message, I think it should be more polite, for example:
"Sorry, there was an unexpected error when trying to process the OPDS feed"

Reply:
ok thanks

Reply:
You changed `error` instead of `reject`. My comment was attached to the `reject` message, which is not accompanied with additional information (as it is a "catch-all" error situation of unknown origin / reason), unlike `error` which has a well-defined `statusCode` and `statusMessage`.

Reply:
done https://github.com/readium/readium-desktop/pull/542/commits/29a4bbccaacd9759bab1f4b1f18913a8a77e29bf